packages/h/hyprland/pspec_x86_64.xml

```diff
@@ -0,0 +1,262 @@
+<PISI>
+    <Source>
+        <Name>hyprland</Name>
+        <Homepage>https://hyprland.org/</Homepage>
+        <Packager>
+            <Name>GENEVEE Nicolas</Name>
+            <Email>gnick@orange.fr</Email>
+        </Packager>
+        <License>BSD-3-Clause</License>
+        <PartOf>desktop.hyprland</PartOf>
+        <Summary xml:lang="en">Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn&apos;t sacrifice on its looks.</Summary>
+        <Description xml:lang="en">Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn&apos;t sacrifice on its looks. It provides the latest Wayland features, is highly customizable, has all the eyecandy, the most powerful plugins, easy IPC, much more QoL stuff than other compositors and more...
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>hyprland</Name>
+        <Summary xml:lang="en">Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn&apos;t sacrifice on its looks.</Summary>
+        <Description xml:lang="en">Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn&apos;t sacrifice on its looks. It provides the latest Wayland features, is highly customizable, has all the eyecandy, the most powerful plugins, easy IPC, much more QoL stuff than other compositors and more...
+</Description>
+        <PartOf>desktop.hyprland</PartOf>
+        <Files>
+            <Path fileType="data">/usr/local/bin/Hyprland</Path>
```

Comment:
This entire package is using the prefix `/usr/local`, which is incorrect (for system packages). It should be `/usr`. Likely this will be fixed by switching to cmake or meson.